frontend/theme/slotRecipes/cardSlotRecipe.ts

```diff
@@ -0,0 +1,19 @@
+import { merge } from "lodash";
+import { defaultConfig, defineSlotRecipe } from "@chakra-ui/react";
+
+const chakraCardSlotRecipe = defaultConfig?.theme?.slotRecipes?.card;
+
+const cardSlotRecipe = defineSlotRecipe(
+  merge(chakraCardSlotRecipe, {
+    base: {
+      root: {
+        borderRadius: 0,
```

Comment:
Use original border radius

```suggestion
        borderRadius: 2,
```